test/src/actions/ios/welcome.actions.js

```diff
@@ -25,6 +26,34 @@ class WelcomeActions extends AndroidWelcomeActions {
         }
         FilesHelper.editJsonByKey(walletData, "masterKey", masterKey);
     }
+
+    async addMasterKey() {
+        for (let i = 0; i < 24;) {
+            for (let j = 1; j <= 3; j++) {
+                await ActionHelper.setText(Pages.welcomePage.typeNemonic(j), importData.importWallet[i]);
+                await ActionHelper.pause(500);
+                if (await $(Pages.welcomePage.wordSuggestion).isDisplayed()) {
```

Comment:
instead of isDisplayed(), try using the wdio-common (wait-helper.js) methods, if the method you need is not there, please request it or open a PR ;)